api/src/application/user/user-controller.js

```diff
@@ -148,6 +148,10 @@ module.exports = (
       const user = request.auth.credentials.user;
       const client = await clientService.findById(request.query.client_id);
 
+      if (!user.get('pending_email_lower')) {
```

Comment:
Heyo - looks ok, but I like to see these checks done with async validation in the route. HMU on slack when you get a sec and I'll walk you through what that looks like w/ a screenshare